predict.py

```diff
@@ -372,6 +372,9 @@ async def predict(
                         f"E2104 TritonMalformedEvent: Triton returned malformed event (no output_ids or error key): {event_data}"
                     )
 
+                if token == []:
+                    continue
+
```

Comment:
was this discovered by testing? what is this?

Reply:
Good question, it was discovered by testing. EOS seems to have been replaced by []. Potentially related to tokenizer config?